src/filing/actions/receiveFiling.js

```diff
@@ -4,7 +4,7 @@ import { isStalledUpload } from '../institutions/helpers'
 
 export default function receiveFiling(data) {
   // Check for stalled uploads
-  data.submissions = data.submissions.map((sub) => {
+  data.submissions = data.submissions?.map((sub) => {
```

Comment:
think we should try to avoid side-effects if we are able to. feels like this one can be done with:
`const filing = {...data, submissions: data.submissions?.map...}` rather than modifying `data.submissions`

Reply:
Good catch, thanks for the keen eye!